examples/hacker_news/src/App.tsx

```diff
@@ -2,29 +2,33 @@ import { Suspense } from 'react'
 import { a, useSpring } from '@react-spring/web'
 import Parser from 'html-react-parser'
 import { Provider, atom, useAtom, useSetAtom } from 'jotai'
+import { atomWithQuery } from 'jotai/query'
```

Comment:
We don't want to change this example with RQ. If you need such a example, please create a new one.